localstack/services/apigateway/helpers.py

```diff
@@ -134,8 +135,8 @@ def get_apigateway_store(context: RequestContext) -> ApiGatewayStore:
 
 
 def get_apigateway_store_for_invocation(context: ApiInvocationContext) -> ApiGatewayStore:
-    account_id = context.account_id or get_aws_account_id()
-    region_name = context.region_name or aws_stack.get_region()
+    account_id = context.account_id or DEFAULT_AWS_ACCOUNT_ID
+    region_name = context.region_name or AWS_REGION_US_EAST_1
```

Comment:
This mimics the existing behaviour. At the time of executing this function, the account ID and region enricher handlers are not executed in the handler chain:

https://github.com/localstack/localstack/blob/f7eff1f5320c4ef35d6a0b88c519ed11431618b5/localstack/aws/app.py#L38

Thus, the `get_aws_account_id()` and `aws_stack.get_region()` return the fallback values.

Reply:
Thanks for the explanation! I hope this doesn't fallback often, I guess it falls back before actually failing but at least we have values. This makes sense and will be revisited in an upcoming APIGW refactor! thanks a lot